idea/api-gateway/src/middleware/check-genesis.middleware.ts

```diff
@@ -0,0 +1,23 @@
+import { Response, Request, NextFunction } from 'express';
+import { IRpcRequest, JSONRPC_ERRORS } from '@gear-js/common';
+
+import { getResponse } from '../utils';
+
+export async function checkGenesisMiddleware(req: Request, res: Response, next: NextFunction) {
+  const body: IRpcRequest = req.body;
+
+  if(Array.isArray(body)) {
+    const isExistGenesis = body.every((value) => 'genesis' in value && value.genesis);
+
+    if (!isExistGenesis) {
+      res.send(getResponse(body, JSONRPC_ERRORS.Forbidden.name));
```

Comment:
Let's make this error more understandable

Reply:
@osipov-mit What would you suggest? To create a new `json-rpc` error or use an already existing `json-rpc` error
message like the bellow ?

 ```javascript 
  {
    name: 'InvalidParams',
    code: -32602,
    message: 'Invalid params',
  }
  ```